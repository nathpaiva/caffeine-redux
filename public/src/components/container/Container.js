import React from 'react';

const container = {
  margin: '5px 0',
  fontSize: '12px'
}

const Container = ({ children }) => (
  <li style={container}>
    {children}
  </li>
);

export default Container;
