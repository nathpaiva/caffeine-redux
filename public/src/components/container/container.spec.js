'use stict';

import React from 'react';
import renderer from 'react-test-renderer';

import Container from './Container';


describe('#Test Container Component', () => {
  it('#Should Container to mach snapshot - li item', () => {
    const tree = renderer.create(<Container><div>Oioi</div></Container>).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
