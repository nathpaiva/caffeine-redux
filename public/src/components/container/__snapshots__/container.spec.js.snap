// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Test Container Component #Should Container to mach snapshot - li item 1`] = `
<li
  style={
    Object {
      "fontSize": "12px",
      "margin": "5px 0",
    }
  }
>
  <div>
    Oioi
  </div>
</li>
`;
